hostfactory/host_provider/src/cyclecloud_provider.py

```diff
@@ -1335,6 +1367,7 @@ def main(argv=sys.argv, json_writer=simple_json_writer):  # pragma: no cover
         logger.debug("Input: %s", json.dumps(input_json))
                 
         if cmd == "templates":
+            logger.info("Using azurecc version %s", version.get_version())
```

Comment:
Great!   Let's also clean up the "Arguments" output lines to make them "BEGIN {operationId} {cmd} - ..." and "END {operationId} {cmd}"